thunder/benchmarks/benchmark_inference.py

```diff
@@ -677,6 +680,7 @@ def parse_args() -> argparse.Namespace:
 
     parser.add_argument("--save-results", action="store_true", help="Save results to JSON file")
     parser.add_argument("--output-dir", type=str, default="./results", help="Directory to save results")
+    parser.add_argument("--attn-implementation", type=str, default=None, help="Attention implementation")
```

Comment:
Would you know what the default is? eager?

https://huggingface.co/docs/transformers/attention_interface

Reply:
I see thank you